sparse/var/lib/nemo-pulseaudio-parameters/algs/mainvolume/btmono

```diff
@@ -0,0 +1,3 @@
+x-nemo.mainvolume.call = "0:-5200,1:-6175,2:-4875,3:-3625,4:-3025,5:-2425,6:-2025,7:-1815,8:-1625,9:-1200,10:-1025,11:-700,12:-575,13:-250,14:-125,15:0"
```

Comment:
I have no idea how these are supposed to work, but it looks suspicious to me that the value for 1 is smaller (-6175) than value for 0 (-5200), while all the other values are monotonically increasing.